deploy/csi-deployment.yaml

```diff
@@ -22,6 +22,7 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       imagePullSecrets:
         - name: regcred
+      initContainers:
```

Comment:
Is this automatically generated? We don't actually have any init containers here.

Reply:
Yes it is, it's empty because jaegertracing is disabled by default